### DISCUSSION TITLE
How does issue tracking work in Git Hub?

### DISCUSSION THREAD

**Discussion by SteveBlower** [musescore/MuseScore#16289] [General]
It is not clear to me how issues progress from being raised to being closed as completed.  In particular, I raised this issue #13678.  at the alpha stage.

The last bit of its history looks like this

![#13678](https://user-images.githubusercontent.com/60138048/217801553-e9e4a83e-7fe1-4b54-95e6-a7147b0f85d2.jpg)

That seems to say it is completed but if I look here https://github.com/musescore/MuseScore/projects/54 I see the issue in the Done column but there is no associated PR or further explanation.

Is this really Done?  Is it ready to be implemented?  Should I question its current status or give a bump so that it does not get lost?

**Comment by ghost** [comment]
For a while, issues which were being bumped to after-4.0 were being closed (temporarily) and put into the 4.X project, just to keep the list of open 4.0 issues manageable. Confusing, true.

When we get to that point in planning we will go through all these closed issues and reopen them and assign them to projects (4.1, the 4,x shortlist, or whatever else we create). You can request that we open specific ones sooner or later if you like - some people have done - but ultimately it doesn't really make that much difference, as we'll be getting to them all in time. Nothing is going to get 'lost'. I agree this limbo state is a little weird though!

The 'Done' status is the result of automation rules which probably don't make sense now either, and we'll be sorting those out as well. (In the absence of any attached PR you can't be pretty sure it is not _really_ 'Done', in any case.)

**Reply by SteveBlower** [reply]
Thanks for the clarification.
